src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOneAndDeleteIntegrationTest.java

```diff
@@ -321,4 +317,151 @@ public void checkMetrics() {
       FindOneAndDeleteIntegrationTest.super.checkMetrics("FindOneAndDeleteCommand");
     }
   }
+
+  @Nested
+  class ConcurrentDelete {
```

Comment:
Good! If these failed (at least for some runs) without the fix, and pass (reliably) with the fix (couple of sample runs should be enough to prove), that's good.